include/swift/Basic/LangOptions.h

```diff
@@ -561,6 +561,9 @@ namespace swift {
     /// Enables dumping type witness systems from associated type inference.
     bool DumpTypeWitnessSystems = false;
 
+    /// Enables `/.../` syntax regular-expression literals
+    bool EnableForwardSlashRegexLiterals = false;
```

Comment:
@hamishknight, anywhere where this can already be used to enable new parsing rules?

Reply:
Not currently no